authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/utils/HttpExceptionUtils.kt

```diff
@@ -37,3 +38,12 @@ fun notFound(message: String = ""): Nothing {
             .build()
     )
 }
+
+fun conflict(message: String = ""): Nothing {
+    throw WebApplicationException(
+        "User already exists. $message",
```

Comment:
I also assume that this exception can be used for more than just users. I suggest either renaming this function to `conflictUser` or using `Resource already exists` as the generic message.